sdk/purview/Azure.Analytics.Purview.Scanning/src/Generated/PurviewClassificationRuleClient.cs

```diff
@@ -360,6 +452,27 @@ public virtual Response Delete(RequestContext context = null)
         /// <param name="action"> The ClassificationAction to use. Allowed values: &quot;Keep&quot; | &quot;Delete&quot;. </param>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="action"/> is null. </exception>
+        /// <example>
+        /// This sample shows how to call TagVersionAsync with required parameters and parse the result.
+        /// <code><![CDATA[
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("<https://my-account-name.azure.com>");
+        /// var client = new PurviewClassificationRuleClient(endpoint, credential);
+        /// 
+        /// Response response = await client.TagVersionAsync(1234, "<action>");
+        /// JsonElement result = JsonDocument.Parse(GetContentFromResponse(response)).RootElement;
+        /// Console.WriteLine(result.GetProperty("scanResultId").ToString());
+        /// Console.WriteLine(result.GetProperty("startTime").ToString());
+        /// Console.WriteLine(result.GetProperty("endTime").ToString());
+        /// Console.WriteLine(result.GetProperty("status").ToString());
+        /// Console.WriteLine(result.GetProperty("error").GetProperty("code").ToString());
```

Comment:
Since we are no longer showing the error response schema in the `<remarks>` block, we should remove parsing error from the examples as well.  In C#, customers get error information via RequestFailedException.

Reply:
This is the `OperationResponse.error` property (see [definition](https://github.com/Azure/azure-rest-api-specs/blob/1c7df99f6a84335cfd7bf5be8c800d72c1dddbc2/specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2018-12-01-preview/scanningService.json#L2019-L2023)) of an LRO [`ClassificationRules_TagClassificationVersion`](https://github.com/Azure/azure-rest-api-specs/blob/1c7df99f6a84335cfd7bf5be8c800d72c1dddbc2/specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2018-12-01-preview/scanningService.json#L427).
It's part of a LRO response, though the operation does define a separate [error response](https://github.com/Azure/azure-rest-api-specs/blob/1c7df99f6a84335cfd7bf5be8c800d72c1dddbc2/specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2018-12-01-preview/scanningService.json#L470-L474).

So, this looks like an issue in the original swagger definition. I'm not sure if we should fix that.

Reply:
Please file an issue to track resolving this in v1.1 or v1.2.  I don't think we need to block GA on this.

Reply:
Filed an issue: https://github.com/Azure/autorest.csharp/issues/2345